### PR TITLE
New topic: dual funding

### PR DESCRIPTION
**Pull request by harding**
- [x] Figure out which BOLTs PR to use (I pinged the author on IRC)

### CODE REVIEW

**Comment by michaelfolkson** [comment]
Definitely worthy of a new topic and looks good. Perhaps a paragraph or two on the challenges of dual funding? I'd need to look into it more and/or speak to the dual funding devs but one challenge I'm aware of is privacy related. A privacy attacker can agree to open a dual funded channel, learn your UTXO(s) and then abort at no cost. So being selective with who you open a dual funded channel with seems like a good idea for now.

**Comment by harding** [comment]
@michaelfolkson added a paragraph hopefully addressing your concern.

---

Also fixed FIXME.  If possible, I'd like to get this merged by Tuesday afternoon so that we can link to it in next Wednesday's newsletter, which mentions dual funding receiving a C-Lightning run-time configuration option.

File: _topics/en/dual-funding.md [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+  want to use the protocol's ability to communicate node preferences and
+  find mutually acceptable values for various channel parameters.
+
+  After dual funding is available, it may be used in combination with
+  new [proposed node announcements][advertising channel liquidity] that
+  could help buyers and sellers of inbound capacity find each other in a
+  decentralized fashion.
+
+  Dual funding does require each party reveal ownership of one of their
+  UTXOs to the other parties.  Like other protocols where this is
```
Comment (jnewbery) [inline _topics/en/dual-funding.md:62]: perhaps s/the other parties/the other party/ . Or maybe "to the proposed counterparty" or similar.

File: _topics/en/dual-funding.md [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+    - title: Dual funding
+      link: https://github.com/lightningnetwork/lightning-rfc/pull/851
+
+## Optional.  Each entry requires "title", "url", and "date".  May also use "feature:
+## true" to bold entry
+optech_mentions:
+  - title: "LN protocol specification 1.1 goals: dual funding"
+    url: /en/newsletters/2018/11/20/#dual-funded-channels
+
+  - title: "Interactive construction of funding transactions (prelude to dual funding)"
```
Comment (jnewbery) [inline _topics/en/dual-funding.md:86]: Consider dropping the "(prelude to dual funding)".

**Review by jnewbery — APPROVED** [review]
Looks great. I've left a couple of tiny suggestions. Feel free to take or ignore. I'll merge afterwards.

**Pull request merged**